packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -33,70 +37,93 @@ describe("pluggable-widgets-tools commands", () => {
         rm(toolsPackagePath);
     });
 
-    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, lang, version) => {
-        const widgetName = `generated_${version.replace(".", "_")}_${lang}_${platform}`;
+    describe.each(CONFIGS)("for %s %s widget created with generator %s", (platform, boilerplate, lang, version) => {
+        const widgetConfigDescription = `generated_${version.replace(".", "_")}_${lang}_${platform}`;
         const workDir = join(tempdir(), `pwt_test_${Math.round(Math.random() * 10000)}`);
+        let widgetPackageJson;
```

Comment:
This variable is not being reassigned anywhere. so I declare here instead of line 75?

Reply:
We need to wait for the project to be generated, before we can assign the file. We need this variable also in another test in the `it`, so that's why it's there.